src/os/query/areamanager.js

```diff
@@ -191,6 +192,26 @@ os.query.AreaManager.mapOriginalGeoms = function(area) {
 };
 
 
+/**
+ * @return {Array<!os.data.AreaNode>} Array of active areas
+ * @export
```

Comment:
The `@export` tag here is unnecessary. That tag causes the compiler to leave the function name the same in compiled mode, which we only need to do for functions referenced by our HTML (and in some other, specialized cases).

I would also recommend moving this function entirely into the areas.js file. The `AreaManager` shouldn't create `AreaNodes` as those are more of a UI construct while this is the model class.